<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>show Vars in MF4</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hbox_load" stretch="1,4">
     <item>
      <widget class="QPushButton" name="btn_loadFile">
       <property name="text">
        <string>LoadFile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labbel_fileName">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MF4 file name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBox_show"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hbox_save" stretch="1,4">
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_version">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mdfreader version:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
